### PR TITLE
 tribute-page

### PR DESCRIPTION
**Pull request by Nooria150**
1. added feature branch
2. setup linters 
3. making pull request
4. and wanna make tribute page with my own design.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+                    Sunshine is my medicine of choice.
+                    Dog days of summer.
+                    Blue skies, sunshine, everything's fine.
+                    Stay salty.
+                    <a id="tribute-link" href="google.com" target="_blank"></a>
+                    </div>
+                </div>
+        </main>
+    </body>
+
```
Comment (ahzamir) [inline index.html]: - Your code missing the closing tag for <html>

File: index.html [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+                    What time is it? Summertime!
+                    Living for these summer days and summer nights.
+                    Good times and tan lines.
+                    Sunshine is my medicine of choice.
+                    Dog days of summer.
+                    Blue skies, sunshine, everything's fine.
+                    Stay salty.
+                    <a id="tribute-link" href="google.com" target="_blank"></a>
+                    </div>
+                </div>
```
Comment (ahzamir) [inline index.html]: - Your code spacing is not aligned correctly:
  - All child elements should be two spaces ahead of their parent element.

**Example:**
```
<div>
  <h1> ... </h1>
  <p>
  .
  .
  </p>
  <div>
    <img ... />
    more elements ...
  </div>
</div>
```

File: styles.css
Comment (ahzamir) [inline styles.css:1]: - [OPTIONAL] - Add some more design to your elements and page so you have a better look to share.

**Review by ahzamir — CHANGES_REQUESTED** [review]
### Hi @Nooria150 🙋‍♀️,

> Excellent work! You have done a great job implementing the project requirements but your project still misses some of the requirements:

- Although your code passes all checks on freecodecamp website on the tribute page project, but I added some inline comments **(Check below)** for you to implement for clean code.

# Status: Required Changes 🔁
Make changes to your code and commit and request for a re-review.

AlmostThereReachGIF

 ## Highlights 🌞
 
🎯 Good Design implemented.
✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
✔️ You have followed and completed the Tribute Page and all requirements.
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)
❌ Your code is clean.

**Cheers 🥂 and Happy coding!!! 👯 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @ahzamir in your question so I can receive the notification.**

**Comment by Nooria150** [comment]
thank you for the review, I implement the feedback. @ahzamir

**Review by ahzamir — APPROVED** [review]
### Hi @Nooria150 🙋‍♀️,

> Excellent work! 💯 You have done a great job implementing all the project requirements. 🥳 Your feature is complete! Congratulations!!!🎊

- **[REQUIRED]** Share your project on this [GROUP](https://www.facebook.com/groups/kick-startdev), add a simple Proud Moment text and a screenshot of your project then besides that add your project link at the end of the text as a GitHub Source code reference and post.

- Share your Project on your LinkedIn account and add a proud moment text on it with a screenshot.

- Well Done great design Implemented. 🎯🎉👍

# Status: Approved ✅
There is nothing to say other than to merge your branch. 🎉🎯✅👍😊🥳
ExcellentNiceGIF

 ## Highlights 🌞

🎉 Great Design Implemented 
✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
✔️ You have followed the Tribute Page project task and committed using the command line
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)

**Cheers 🥂 and Happy coding!!! 👯 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @ahzamir in your question so I can receive the notification.**

**Comment by Nooria150** [comment]
thank you for approving.

**Pull request merged**